addons/web/static/src/views/graph/graph_renderer.js

```diff
@@ -15,7 +22,107 @@ import { ReportViewMeasures } from "@web/views/view_components/report_view_measu
 
 const NO_DATA = _t("No data");
 
-export const LINE_FILL_TRANSPARENCY = 0.4;
+const colorScheme = cookie.get("color_scheme");
+const GRAPH_LEGEND_COLOR = getCustomColor(colorScheme, "#111827", "#ffffff");
+const GRAPH_GRID_COLOR = getCustomColor(colorScheme, "rgba(0,0,0,.1)", "rgba(255,255,255,.15");
+const GRAPH_LABEL_COLOR = getCustomColor(colorScheme, "#111827", "#E4E4E4");
+
+/**
+ * Custom Plugin for Line chart:
+ * Draw the scale grid on top of the chart to
+ * see this last one correctly.
+ */
+const gridOnTop = {
+    id: "gridOnTop",
+    afterDraw: (chart, args, options) => {
+        if (chart.config.options.plugins.gridOnTop) {
+            const elements = chart.getDatasetMeta(0).data || [];
+            const ctx = chart.ctx;
+            const chartArea = chart.chartArea;
+            const yAxis = chart.scales.y;
+            const xAxis = chart.scales.x;
+
+            ctx.lineWidth = options.gridLineWidth;
```

Comment:
Can't you hard code  what you want here instead of passing options for you plugin in lines after 381?